src/powfacpy/base_interface.py

```diff
@@ -1003,8 +1003,27 @@ def get_name_with_ending(objects):
     if not type(objects) == list:
       objects = [objects]
     return [x.GetFullName().split('\\')[-1] for x in objects]
-    
 
-if __name__ == "__main__":
-  print("ok")
-  
\ No newline at end of file
+
+def set_attr_of_obj(obj, attributes:dict):
```

Comment:
no abbreviatons in naming -- better: 'set_attributes_of_object'

Reply:
attr is a very common abbreviation in OO programming. There is also a python method named setattr, so this abbreviation is ok I think. obj should also be clear and is used in many method names in powfacpy.